packages/next/src/server/lib/patch-fetch.ts

```diff
@@ -460,16 +461,18 @@ export function patchFetch({
                   staticGenerationStore.pendingRevalidates.push(
                     Promise.resolve(originalFetchResponse)
                   )
-                  // if the request is stale and it's the first request that will miss the cache
-                  // this ensure the response will come with the new data requested
-                  const responseBodyArrayBuffer =
-                    await originalFetchResponse.arrayBuffer()
-                  const base64String = btoa(
-                    String.fromCharCode(
-                      ...new Uint8Array(responseBodyArrayBuffer)
+                  if (forceFreshOnStale) {
```

Comment:
I suggest staleWhileRevalidate?: boolean (default true). Docs need to be updated as well.